examples/src/main/python/sql/udtf.py

```diff
@@ -39,13 +39,13 @@ def eval(self, start: int, end: int):
             for num in range(start, end + 1):
                 yield (num, num * num)
 
-    from pyspark.sql.functions import lit, udtf
+    from pyspark.sql.functions import udtf
 
     # Create a UDTF using the class definition and the `udtf` function.
     square_num = udtf(SquareNumbers, returnType="num: int, squared: int")
 
     # Invoke the UDTF in PySpark.
-    square_num(lit(1), lit(3)).show()
+    square_num(1, 3).show()
```

Comment:
dumb question: in case a input is `str`, e.g.

```
square_num(`a`, 3)
```

should we treat `a` as a literal string or a column name (like most functions) ?

Reply:
+1, the string input might be tricky to decide